modules/ROOT/pages/6.3-release-notes.adoc

```diff
@@ -0,0 +1,716 @@
+* xref:accompanying-premium-plugin-changes[Accompanying Premium Plugin changes]
+* xref:accompanying-premium-skins-and-icon-packs-changes[Accompanying Premium Skins and Icon Packs changes]
+* xref:improvements[Improvements]
+* xref:additions[Additions]
+* xref:changes[Changes]
+* xref:bug-fixes[Bug fixes]
+* xref:security-fixes[Security fixes]
+* xref:known-issues[Known issues]
+
+[[new-premium-plugins]]
+== New Premium plugins
+
+=== Inline CSS 1.0.0
+
+The {productname} 6.3 release includes the new **Inline CSS** premium plugin.
+
+The Inline CSS plugin allows the user to <notes here>.
+
+For information on the **Inline CSS** plugin, see: xref:inline-css.adoc[Inline CSS].
+
+=== Advanced Typography 1.0.1
+
+The {productname} 6.3 release includes the new **Advanced typography** premium plugin.
+
+The **Advanced Typography** plugin, depending on its setting, can replace common typewriter-style characters with their professional typesetter equivalents after the _Advanced Typography_ button is pressed. This action will modify either the entire content for the {productname} document or the specific text selection highlighted by the user.
+
+For information on the **Advanced typography** plugin, see: xref:advanced-typography.adoc[Advanced typography].
+
+There are several known issues in **Advanced Typography** 1.0.0:
+
+==== Switching the Advanced Typography target language does not always work as expected
+
+Selecting a language from the *Advanced Typography* toolbar button’s menu, runs the rules for the newly selected over the current {productname} document.
+
+In some circumstances, switching the target language in this fashion does not result in typesetting changes as expected. This is most visibly apparent with quotation marks.
+
+The British English typesetting norm is to use opening and closing single-quote marks to denote direct quotation: *‘* and *’*.
+
+The American English typesetting norm is to use opening and closing double-quote marks to denote direct quotation: *“* and *”*.
+
+And one Spanish typesetting norm is to use guillemets, also known as comillas angulares, comillas franceses or chevrons, to denote direct quotation: *«* and *»*.
+
+So, for example, if a {productname} document included a string thus:
+
+[source,text]
+----
+"They went that way," he said, pointing.
+----
+
+Selecting _English (US)_ from the *Advanced Typography* toolbar icon’s pop-up menu transforms the text, as expected, into
+
+[source,text]
+----
+“They went that way,” he said, pointing.
+----
+
+Following this action by, then, selecting _Spanish_ from the *Advanced Typography* toolbar icon’s pop-up menu, however, can result in the following incomplete typesetting change:
+
+[source,text]
+----
+«They went that way, ” he said, pointing.
+----
+
+As of this release the only workaround is to not change *Advanced Typography* target languages.
+
+==== The Advanced Typography plugin erroneously honors the current SpellChecker Pro language selection
```

Comment:
@brian-forte Please remove this issue from list as this is fixed now by this ticket. Its verified by QA. https://ephocks.atlassian.net/browse/TINY-9413

Reply:
To quote my commit message:

> 1. Removed known issue that has been fixed and is no longer a known issue.
> 2. Corrected version number in the body of the Advanced Typography section.
> 3. Edited the intro sentence to the Advanced Typography known issues sub-section: there is only one known issue so switched said sentence from plural to singular.